contracts/Tokens/VTokens/VTokenInterfaces.sol

```diff
@@ -114,6 +131,32 @@ contract VTokenStorage {
      * @notice Mapping of account addresses to outstanding borrow balances
      */
     mapping(address => BorrowSnapshot) internal accountBorrows;
+
+    /**
+     * @notice Model which tells what the current stable interest rate should be
+     */
+    StableRateModel public stableRateModel;
+
+    /**
+     * @notice Total amount of outstanding stable borrows of the underlying in this market
+     */
+    uint256 public stableBorrows;
+
+    /**
+     * @notice Accumulator of the total earned stable interest rate since the opening of the market
+     */
+    uint256 public stableBorrowIndex;
+
+    /**
+     * @notice Average of all of the stable borrows
+     */
+    uint256 public averageStableBorrowRate;
+
+    // Maximum stable borrow rate that can ever be applied (.0005% / block)
+    uint256 internal constant stableBorrowRateMaxMantissa = 0.0005e16;
```

Comment:
isn't mantissa 1e18? so in this case shouldn't this value be `0.0005e18` ?

Reply:
If we want to write in the 1e18 then we have to keep as `0.000005e18` instead of `0.0005e16`. But for better readability it is kept in that way.

Reply:
yeah fair enough :)